### DISCUSSION TITLE
Application is denied due to no expiration date. My ID doesn't have one.

### DISCUSSION THREAD

**Discussion by astrobc1** [community/community#139985] [GitHub Education]
### Hi everyone,

I'm an educator and would like access to GitHub Copilot, but my ID doesn't have an expiration date. So my application was denied. What other documents are valid?

**Comment by davevad93** [comment]
Hi @astrobc1 ,

The [docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student) state the following:

- ***Documents that prove your current student status include a picture of your school ID with current enrollment date, class schedule, transcript, and affiliation or enrollment verification letter.***

You can provide any of the above documents with your full name, the school's name and a current date in a `DD/MM/YYYY`, `MM/DD/YYYY` or `YYYY/MM/DD` format.

The last time I applied, I uploaded a screenshot of my last exam results with the above requirements and I was approved.

EDIT: Sorry I misread and didn't notice that you're an educator, so it's not the same as applying as a student; I don't know if you could provide other documentation as if you were a student but you could try anyway.
